app/assets/stylesheets/styles.css.scss

```diff
@@ -0,0 +1,45 @@
+
+body {
+  background: transparent url("texture.png");
+}
+
+#nav_bar {
+  color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Properties should be ordered background-color, color